src/util/pg.py

```diff
@@ -362,14 +362,26 @@ def explode_execute(cr, query, table, alias=None, bucket_size=10000, logger=_log
     )
 
 
+class SQLStr(sql.SQL, str):
+    def __add__(self, other):
```

Comment:
It needs a meta:private docstring or proper docstrings :/ (the latter is better)
![image](https://github.com/user-attachments/assets/7f7ec864-f0f9-4d38-89a4-0197319f5696)